packages/-build-infra/src/node-test-helpers/nodetest-runner.js

```diff
@@ -0,0 +1,81 @@
+'use strict';
```

Comment:
I don't think we _need_ this file any more, we should remove (seems fine to do as a follow up).

Reply:
@rwjblue why wouldn't we need the test runner anymore?

Reply:
What does it do that the `mocha` script itself doesn't? We originally (waaaaaaaaayyyyyy back) needed it because `mocha` didn't support globs properly, I think we can remove it.

Reply:
if we shift to a lint rule for no `.only` and fix our windows support we *might* be able to remove it? In addition to those things and glob it seems to be ensuring some form of a temporary node_modules directory, unsure if that directory is even in use anymore. Will open an issue.

Reply:
https://github.com/emberjs/data/issues/6111